pkg/util/hostport/hostport.go

```diff
@@ -65,3 +65,42 @@ func RevertPorts(replacementPortsMap, originalPortsMap map[LocalPort]Closeable)
 		}
 	}
 }
+
+// OpenLocalPort opens the given host port and hold it.
```

Comment:
Nit: "OpenLocalPort opens the given port on the node and holds it". Since the function is used in both hostPort and nodePort situations. I think the comment should be more generic.